data/bash-completion/fwupdtool.in

```diff
@@ -109,7 +109,7 @@ _fwupdtool()
 			_show_modifiers
 		fi
 		;;
-	attach|detach|activate|verify-update|reinstall)
+	attach|detach|activate|verify-update|reinstall|get-updates)
```

Comment:
can you also update the fwupdmgr bash completion file for the same change?

Reply:
Done